header.go

```diff
@@ -37,6 +38,10 @@ func newHeader(cards []Card, htype HDUType, bitpix int, axes []int) *Header {
 
 // NewHeader creates a new Header from a set of Cards, HDU Type, bitpix and axes.
 func NewHeader(cards []Card, htype HDUType, bitpix int, axes []int) *Header {
+	// short circuit: too many axes violates the FITS standard
+	if len(axes) > 999 {
+		panic(fmt.Sprintf("len(axes) is %d, the FITS standard only allows 999 dims", len(axes)))
```

Comment:
```suggestion
		panic(fmt.Errorf("fitsio: too many axes (got=%d > 999)", len(axes)))
```

also, we could perhaps add a test that triggers that panic (and make sure the panic error is what we expect)

Reply:
added a test for panic -- I don't know how to (easily) get at the panic text after recover